src/editors/analyticsEvt.js

```diff
@@ -0,0 +1,6 @@
+export const analyticEvt = {
```

Comment:
Might be worth putting this under data/constants/, in order to have it near all our other constants, but I'm open to whatever.

Reply:
I was modeling it after messages.jsx, but I like this more! Will move.